msg/RadarTrack.msg

```diff
@@ -0,0 +1,23 @@
+# This message relates only to FMCW radar.  
+# All variables below are relative to the radar's frame of reference.
+# This message is not meant to be used alone but as part of a stamped or array message.
+
+# Object classifications (Additional vendor-specific classifications are permitted eg. 3 = Car)
```

Comment:
I generally don't like leaving these open. But definitely don't suggest the usage of 3. This should be expanded to maybe a uint16 and reserve the upper half for additional vendor specific classifications and leave the lower half for potential future standardization. 

On the scale of this message optimizing 8 bit vs 16 bit is not worth it.

Reply:
Agreed. @radarAaron I think the action items here are:
- change uint8 to uint16 for all pre-defined classifications & `classification`.
- Keep `car` but remove `3 =` since that implicitly sets 3 as a car, which I don't think was the intent. 

I would suggest we could reserve the first `100` for further standardization rather than 32,000 - that seems excessive. Thoughts?

Reply:
> I would suggest we could reserve the first `100` for further standardization rather than 32,000 - that seems excessive.

In my experience you end up never having enough of anything. Since you're already going to `uint16`, *really* future-proof it and do what @tfoote suggests.

Users won't care whether OEM ids start at 100 or 32000, it's just a number.

Reply:
I agree with @gavanderhoorn - Let's start vendor-specific values soemwhere around 32000.

Reply:
Changes made as requested.

Reply:
@radarAaron one last part of that - you need to update the `uint8`'s for static, uknown, and dynamic.

Reply:
done